Encodings.md

```diff
@@ -61,7 +61,8 @@ followed by the values encoded using RLE/Bit packed described above (with the gi
 Using the PLAIN_DICTIONARY enum value is deprecated in the Parquet 2.0 specification. Prefer using RLE_DICTIONARY
 in a data page and PLAIN in a dictionary page for Parquet 2.0+ files.
 
-### <a name="RLE"></a>Run Length Encoding / Bit-Packing Hybrid (RLE = 3)
+<a name="RLE"></a>
```

Comment:
Is moving the anchor out of the header an actual fix or just formatting? Asking only because this is how all the rest of the headers are.

Reply:
It's just a fixing for heading anchor in current format